managed/ui/src/components/config/PublicCloud/ProviderConfigurationContainer.js

```diff
@@ -130,6 +132,8 @@ const mapDispatchToProps = (dispatch) => {
     deleteProviderConfig: (providerUUID) => {
       dispatch(deleteProvider(providerUUID)).then((response) => {
         if (response.payload.status !== 200) {
+          const errorMessage = response.payload?.response?.data?.error;
+          toast.error(errorMessage)
```

Comment:
`errorMessage` could be undefined if there's no `response.payload?.response?.data?.error` and empty toast will popup

Reply:
same as above we can have a generic error message for such cases, need to come up with a meaningful sentence.